pkg/machines/machines.less

```diff
@@ -270,14 +270,6 @@ table.vcpu-detail-modal-right .btn-group.bootstrap-select.dropdown {
     display: inline-block;
 }
 
-/* FIXME: This CSS should be moved upstream */
-.form-text {
-    /* Set size similar to a widget; like control-label, but no alignment */
-    display: inline-block;
-    padding-top: 3px;
-    margin-bottom: 0;
-}
-
```

Comment:
There is reference of this class here.
pkg/machines/components/nicEdit.jsx:            <samp className='form-text' id='mac'>

But I can remove it in the future. No harm.